packages/react-scripts/template/src/utils/Request.js

```diff
@@ -0,0 +1,66 @@
+import URI from 'urijs';
+import axios from './axios';
+
+/**
+ * Fluent interface to encapsulate application-specific REST request logic
+ * HTTP-based methods proxy to [axios](https://github.com/mzabriskie/axios#request-method-aliases) to execute the
+ * request. The others are backed by [URIjs](https://medialize.github.io/URI.js) to build URI, which is used by the
+ * HTTP methods.
+ *
+ * Example:
+ *  const promise = new Request().pod().genericQuery('folders).post();
```

Comment:
Update comment, there is no `pod()`

Reply:
I left it to indicate it should be customized however it was needed.